src/components/Dashboard/TodoList.vue

```diff
@@ -890,6 +952,10 @@ export default {
       }
     },
 
+    isBcolError (task: any): boolean {
+      return task.paymentErrorType && task.paymentErrorType.startsWith('BCOL')
+    },
```

Comment:
FYI, you can also now write this as
`return task.paymentErrorType?.startsWith('BCOL')`

Reply:
ah like C# :D

Reply:
Yes, like that. After all, they're both from Microsoft :)

Reply:
Code was changed. Comment no longer applies.